.github/workflows/one-click-hosting-release.yml

```diff
@@ -0,0 +1,138 @@
+      - name: Get Configuration
+        id: configuration
+        uses: maork-elementor/json-to-variables@main
+        with:
+          filename: './.github/workflows/config.json'
+          prefix: config
+      - name: Test permissions
+        uses: ./.github/workflows/permissions
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY_OWNER: ${{ env.config_deployment_repository_owner }}
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Is pre-release?
+        id: is_pre_release
+        run: |
+          if [ ${{ github.event.inputs.pre_release }} == true ]; then
+            PRE_RELEASE="-prerelease"
+            echo "This is a prerelease execution"
+          else
+            PRE_RELEASE=""
+            echo "This is a release execution"
+          fi
+          echo "PRE_RELEASE=${PRE_RELEASE}" >> $GITHUB_ENV
+      - name: Env Vars Configure
+        uses: ./.github/workflows/env-vars-configure
+        with:
+          CHANNEL: ${{ env.CHANNEL }}
+          REF: ${{ github.ref }}
+          ADDITION_NAME: ${{ env.PRE_RELEASE }}
+      - name: Bump Channel Version
+        uses: ./.github/workflows/bump-channel-version
+        with:
+          CLEAN_PACKAGE_VERSION: ${{ env.CLEAN_PACKAGE_VERSION }}
+          CHANNEL: ${{ env.CHANNEL }}
+          OVERRIDE_PACKAGE_VERSION: true
+          ADDITION_NAME: ${{ env.PRE_RELEASE }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+      - name: Generate changelog
+        uses : ./.github/workflows/generate-changelog
+        with:
+          TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
+          REPOSITORY: ${{ github.repository }}
+          HEAD_BRANCH_NAME: ${{ github.ref }}
+          BASE_TAG_NAME: ${{ env.PREVIOUS_TAG_SHA }}
+          GENERATE_EMPTY_CHANGELOG: true
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ env.PACKAGE_VERSION }}
+          target_commitish: ${{ env.CLEAN_REF }}
+          files: |
+            elementor-*.zip
+            ${{ env.CHANGELOG_FILE }}
+          # Will be removed once we see that the release is working
+          prerelease: ${{ github.event.inputs.pre_release }}
+          body_path: ${{ env.CHANGELOG_FILE }}
+      - name: Post To Slack Created Cloud one click Release
+        if: ${{ github.event.inputs.pre_release }} == false
+        uses : ./.github/workflows/post-to-slack
+        with:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_TOKEN }}
+          SLACK_TAG_CHANNELS: ${{ secrets.SLACK_CHANNEL_RELEASEBOT }}
+          PAYLOAD: |
+            {
```

Comment:
Can we improve this after this PR by loading the Json from a json file?

```suggestion
            `$(cat ./payload.json)`
```

Reply:
Done

Reply:
Unfortunately, after many tryes this can't be done at the moment.